sqrl-calcite/src/main/java/com/datasqrl/plan/validate/ScriptPlanner.java

```diff
@@ -256,6 +250,9 @@ public Void visit(SqrlExportDefinition node, Void context) {
               exportRelNode.getRowType(), List.of(), LogFactory.Timestamp.NONE);
           tableSink = sinkLog.getSink();
         }
+      } else if (connector == SystemBuiltInConnectors.LOG_ENGINE) {
+        Log sinkLog = logManager.getLogs().get(sinkPath.getLast().getDisplay());
+        tableSink = sinkLog.getSink();
```

Comment:
NPE

Reply:
How? builtInSink present in this case, therefore we have a connector.

Reply:
getLogs is a hashmap. We should fall back to generating a log engine (like you were describing earlier)